src/ray/core_worker/actor_manager.h

```diff
@@ -172,6 +180,14 @@ class ActorManager {
   void HandleActorStateNotification(const ActorID &actor_id,
                                     const rpc::ActorTableData &actor_data);
 
+  /// Function that's invoked when the actor is out of scope.
+  ///
+  /// \param actor_handle The actor handle that will be marked as invalidate.
+  void MakeActorInvalid(std::shared_ptr<ActorHandle> actor_handle);
```

Comment:
```suggestion
  void InvalidateActor(std::shared_ptr<ActorHandle> actor_handle);
```

Can you also expand the doc a bit more to say what it means to be "valid"? E.g., "after the actor is invalidated, task submission to the actor will throw an exception" or something like that.